docs/_includes/Common-Commands/copy-alluxio-site-properties.md

```diff
@@ -1,3 +1,3 @@
 ```bash
-$ cp $ALLUXIO_HOME/conf/alluxio-site.properties.template ~/.alluxio/alluxio-site.properties
-```
\ No newline at end of file
+$ cp $ALLUXIO_HOME/conf/alluxio-site.properties.template /etc/alluxio/alluxio-site.properties
+```
```

Comment:
The user may not have access to `/etc/`, perhaps we should have both putting it in the conf directory and `/etc/` and say `/etc/` will take effect even in nonAlluxio processes?